src/logging.rs

```diff
@@ -7,15 +8,52 @@ use std::fs::OpenOptions;
 use std::io::Write;
 use std::os::unix::prelude::FromRawFd;
 use std::path::PathBuf;
+use std::string::{String, ToString};
 use std::sync::Mutex;
 use time::format_description::well_known::Rfc3339;
 use time::OffsetDateTime;
 
-enum LogFormat {
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
+pub enum LogFormat {
 	Text,
 	Json,
 }
 
+impl Default for LogFormat {
+	fn default() -> LogFormat {
+		LogFormat::Text
+	}
+}
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
+pub enum LogLevel {
+	Info,
+	Warn,
+	Debug,
+	Trace,
+	Error,
+	Off,
+}
```

Comment:
Could we use [`log::LevelFilter`](https://docs.rs/log/latest/log/enum.LevelFilter.html) directly?

Reply:
I think that this isn't possible. Clap's parser doesn't support it and I need to derive with the entry `ValueEnum`.

```
#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
#pub enum LogLevel
```